.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,48 @@
+---
+name: 🐛 Bug Report
+about: If something isn't working as expected 🤔.
+
+---
+
+## Bug Report
+<!--
+Thank you for reporting an issue.
+
+Please fill in as much of the template below as you're able.
+-->
+
+### Version & Features
+
+<!--
+List the versions and features of any `kube` and `k8s-openapi` crates you are using.
+Output of `grep kube Cargo.toml`
+Output of `grep k8s-openapi Cargo.toml`
```

Comment:
Probably want to look at Cargo.lock instead (since that contains the actual versions used). But that is a bit more annoying to parse...

Reply:
Yeah makes sense.